test/gibbs.jl/gibbs_constructor.jl

```diff
@@ -24,14 +24,14 @@ c4 = sample(gdemo(), s4)
 c5 = sample(gdemo(), s5)
 
 # Very loose bound, only for testing constructor.
-for c in [c1, c2, c3 ,c4, c5]
-  check_numerical(c, [:s, :m], [49/24, 7/6], eps=1.0)
-end
+#for c in [c1, c2, c3 ,c4, c5]
```

Comment:
I suppose this is work in progress. Maybe add something to ensure that this changes will not move into the master.

Reply:
Yes I will double check all comments before making this camera-ready.